From 6cc54131c41f4aa362dc5cf9f1e7c12d1578b6bd Mon Sep 17 00:00:00 2001
From: Haohui Mai <wheat9@apache.org>
Date: Sat, 3 Oct 2015 11:44:34 -0700
Subject: [PATCH 1071/2848] HDFS-9193. Fix incorrect references the usages of
 the DN in dfshealth.js. Contributed by Chang Li.

(cherry picked from commit 460b5db9ffc00cf2455e59fbfe88621ee94c7f4b)

Change-Id: I9c5180748f0f7ca3591d00fee6060435ba623fa1
---
 .../hadoop-hdfs/src/main/webapps/hdfs/dfshealth.js |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.js b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.js
index 2069619..fd51b68 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.js
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/hdfs/dfshealth.js
@@ -186,7 +186,7 @@
         var r = null;
         if (v < 70) {
           r = 'progress-bar-success';
-        } else if (u.usedPercentage < 85) {
+        } else if (v < 85) {
           r = 'progress-bar-warning';
         } else {
           r = "progress-bar-danger";
-- 
1.7.9.5

